docs/Logging/index.md

```diff
@@ -62,9 +62,23 @@ class ClassTwo
 }
 ```
 
+If you want to programmatically disable logging for methods or classes it can be done via passing a boolean as the first parameter of the trace function. 
+
+```csharp
+class ClassTwo
+{
+	static void MethodFour(string variableOne, int variableTwo)
+	{
+		auto trace = CF_Trace_2("ClassTwo", "MethodFour").Add(variableOne).Add(variableTwo);
+	}
+}
+```
+
```

Comment:
I think you wanted to not copy paste the example above and add the variable you were talking about here